src/NHibernate.Test/TransactionTest/TransactionFixture.cs

```diff
@@ -1,19 +1,13 @@
 using System;
-using System.Collections;
-using System.Data.Common;
+using System.Linq;
+using NHibernate.Linq;
 using NUnit.Framework;
 
 namespace NHibernate.Test.TransactionTest
 {
 	[TestFixture]
-	public class TransactionFixture : TestCase
+	public class TransactionFixture : TransactionFixtureBase
 	{
-		protected override IList Mappings
-		{
-			// The mapping is only actually needed in one test
-			get { return new string[] {"Simple.hbm.xml"}; }
```

Comment:
This `Simple` entity is anything but simple. It does not have a mapped id property, just a column in database, causing weird patterns when actually willing to use it.